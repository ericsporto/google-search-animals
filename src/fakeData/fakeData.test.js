import { faker } from '@faker-js/faker';
import { afterEach, describe, expect, it } from 'vitest';

afterEach(() => {
  faker.seed();
});

describe('reverse array', () => {
  it('should reverse the array', () => {
    faker.seed(1234);
    const title = faker.person.jobTitle();
    const description= faker.lorem.sentence();
    const animal = faker.animal.bear();

    const array = [title, description, animal];

    expect(array.reverse()).toStrictEqual([animal, description, title]);

    expect(array.reverse()).toMatchSnapshot();
  });
});