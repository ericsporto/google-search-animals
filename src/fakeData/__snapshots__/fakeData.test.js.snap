// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`reverse array > should reverse the array 1`] = `
[
  "Dynamic Optimization Orchestrator",
  "Debitis perspiciatis territo tepidus termes undique certe arcus charisma.",
  "Spectacled bear",
]
`;
